streamingcli/main.py

```diff
@@ -83,75 +76,28 @@ def project_init(
 
 @project.command()
 @click.option("--docker-image-tag", help="Docker image tag", default="latest")
+@click.option("--env", help="Environment name to use")
 @click.option(
-    "--docker-image-registry",
-    "docker_image_registry",
-    help='URL for Docker registry, i.e: "https://hub.docker.com/"',
-)
-@click.option(
-    "--docker-image-repository",
-    "docker_image_repository",
-    help="Docker image repository",
-)
-@click.option("--profile", help="Profile name to use")
-@click.option(
-    "--deployment-mode",
-    "deployment_mode",
-    type=click.Choice([x.name for x in DeploymentMode], case_sensitive=False),
-)
-@click.option(
-    "--vvp-url",
-    "ververica_url",
-    help='URL for Ververica cluster, i.e: "https://vvp.streaming-platform.example.com"',
-)
-@click.option("--vvp-namespace", "ververica_namespace", help="Ververica namespace")
-@click.option(
-    "--vvp-deployment-target",
-    "ververica_deployment_target_name",
-    help="Ververica deployment target name",
-)
-@click.option(
-    "--vvp-deployment-template-path",
-    "ververica_deployment_template_path",
-    help="Optional custom Ververica deployment descriptor file absolute path",
+    "--file-descriptor-path",
+    "file_descriptor_path",
+    help="Optional custom deployment descriptor file path",
```

Comment:
Do we still need that? I thought we are following conventions here so descriptor will be in `config` catalog.

What will happen if optional is not provided?